app/controllers/application_controller.rb

```diff
@@ -27,6 +27,21 @@ def current_user_uuid
   end
 
 
+  def current_device_uuid
+    @current_device_uuid ||= begin
+      if Rails.env.development? && ENV['STUBBED_DEVICE_UUID']
+        ENV['STUBBED_DEVICE_UUID']
+      else
+        if ENV['STUBBED_DEVICE_UUID']
+          Rails.logger.warn("`STUBBED_DEVICE_UUID` environment variable is set but not used in " \
```

Comment:
Will this be a lot of extra logging in test runs?  In production we won't care, will we care in test?

Reply:
agree. imo, it's not that helpful, let's delete